src/main/java/com/google/sps/tool/GoogleTranslate.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.tool;
+
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.TranslateOptions.Builder;
+import com.google.cloud.translate.Translation;
+
+public class GoogleTranslate {
+
+  public static String translateText(String textNotTranslated, String targetLanguage) {
```

Comment:
`textNotTranslated` -> `originalText`?

Right now this method is a bit hard to read with so many "translated" words.